src/util/namespace.h

```diff
@@ -84,8 +84,14 @@ class namespacet:public namespace_baset
 
   using namespace_baset::lookup;
 
-  // these do the actual lookup
+
+  /// Searches for a symbol named \p name in `symbol_table1` and returns a
+  /// pointer to it in output parameter \p symbol when found. When no symbol is
+  /// found, it performs the same lookup in `symbol_table2`.
```

Comment:
Neither `symbol_table1` nor `symbol_table2` are part of the interface. Thus I'd refrain from using it in this explanation. More importantly, the second parameter is not explained.